apps/andi/assets/css/datasets.scss

```diff
@@ -6,6 +6,10 @@
   font-size: 1.5em;
 }
 
+.datasets-search {
+  margin: 0.5em;
```

Comment:
We're going to make these all rems, but Brewer will clean up any of the old ones on his card.

Reply:
Is REM going to be properly responsive to different screen sizes/resolutions?

Reply:
Advice from https://zellwk.com/blog/rem-vs-em/:

1. Size in em if the property scales according to its font-size
2. Size everything else in rem.

So it seems like rems are a good choice here.